pkg/controller/group/group_controller.go

```diff
@@ -314,6 +318,15 @@ func (r *GroupReconciler) filterEndpointGroupsByEndpoint(ctx context.Context, en
 	endpointNamespaceLabels = endpointNamespace.Labels
 
 	for _, group := range groupList.Items {
+		// Only SecurityPolicy's named port feature need all-endpoins group,
+		// so if endpoint doesn't define named port, it doesn't need to related to the group.
+		if group.Name == constants.AllEndpointsGroupName {
```

Comment:
AllEndpoints may be confused with "whole endpoints set", AllEpWithNamedport ?

Reply:
or add a special label on endpoint, then the group could be a label selector?

Reply:
It is not a real label, so I think use name "AllEpWithNamedport" is better

Reply:
done